WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/PostBookmarkActionUseCase.kt

```diff
@@ -0,0 +1,23 @@
+package org.wordpress.android.ui.reader.repository.usecases
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import org.wordpress.android.models.ReaderPost
+import org.wordpress.android.modules.IO_THREAD
+import org.wordpress.android.ui.reader.actions.ReaderPostActionsWrapper
+import javax.inject.Inject
+import javax.inject.Named
+
+class PostBookmarkActionUseCase @Inject constructor(
```

Comment:
We have `ReaderPostBookmarkUseCase` and this name seems quite similar. How would you feel about renaming it to something like "UpdateBookmarkStateUseCase"?